### PR TITLE
US75367 - filter out everything except GD Finding cwe detail type

### PR DESCRIPTION
**Pull request by ikemsley**
**Problem**

It's possible that CT events are sent to the cwe collector, but they get 400's from ingest.

**Solution**

Ensure that the cwe collector filters out everything other than "detail-type": "GuardDuty Finding".

### CODE REVIEW

**Comment by ikemsley** [comment]
@kkuzmin Please could you give +1 after testing.  There was an issue with checkin failing because orfthe order of the GD event rule pattern and a space that was giving trouble.  Seems to be good now.

File: cfn/guardduty.template [hunk 1]
```diff
@@ -1134,7 +1137,7 @@
                                    ]
                                 },
                                 "\", \"CloudWatchEventsRule\": \"", { "Ref": "CloudWatchEventsRule" },
-                                "\", \"CweRulePattern\": \"{\\\"source\\\":[\\\"aws.guardduty\\\"]}\"",
+                                "\", \"CweRulePattern\": \"{\\\"detail-type\\\":[\\\"GuardDuty Finding\\\"],\\\"source\\\":[\\\"aws.guardduty\\\"]}\"",
```
Comment (kkuzmin) [inline cfn/guardduty.template]: If we make this change then all existing collectors will report error status.

Comment (kkuzmin) [reply cfn/guardduty.template]: or actually not. Should be ok since this is an env variable set via a template.

Comment (ikemsley) [reply cfn/guardduty.template]: @kkuzmin From testing it seems to be okay.  My stack yesterday was reverted back to the master version of the lambda function so this template even works with the current lambda function, afaics.

**Pull request merged**